### PR TITLE
remove unused codes

### PR DESCRIPTION
**Pull request by SaeedNoshadi89**
Related to #104

### CODE REVIEW

**Comment by SaeedNoshadi89** [comment]
@JolandaVerhoef I removed unused codes to cleanup the project

File: feature-topic/src/main/java/com/google/samples/apps/nowinandroid/feature/topic/TopicScreen.kt [hunk 1]
```diff
@@ -174,8 +172,6 @@ private fun LazyListScope.TopicCards(news: NewsUiState) {
             newsResourceCardItems(
                 items = news.news,
                 newsResourceMapper = { it },
-                isBookmarkedMapper = { /* TODO */ false },
```
Comment (tunjid) [inline feature-topic/src/main/java/com/google/samples/apps/nowinandroid/feature/topic/TopicScreen.kt:177]: You shouldn't remove these, they're todo markers for future feature work.

Comment (SaeedNoshadi89) [reply feature-topic/src/main/java/com/google/samples/apps/nowinandroid/feature/topic/TopicScreen.kt]: @tunjid Sure, I reverted these parts.

**Comment by JolandaVerhoef** [comment]
Hi @SaeedNoshadi89. Can you please check your failing build?

**Comment by SaeedNoshadi89** [comment]
> Hi @SaeedNoshadi89. Can you please check your failing build?

Done. It's ok now

File: core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCardList.kt [hunk 2]
```diff
@@ -53,8 +51,6 @@ fun <T> LazyListScope.newsResourceCardItems(
 
         NewsResourceCardExpanded(
             newsResource = newsResource,
-            isBookmarked = isBookmarkedMapper(item),
```
Comment (calren) [inline core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCardList.kt:56]: we should keep `isBookmarked` and `onToggleBookmark` params. these are upcoming features in progress

**Comment by JoseAlcerreca** [comment]
Thanks for the PR, but I don't think it should be merged. Most things are WAI.

File: build.gradle.kts [hunk 3]
```diff
@@ -26,5 +26,6 @@ buildscript {
         classpath(libs.kotlin.serializationPlugin)
         classpath(libs.hilt.gradlePlugin)
         classpath(libs.secrets.gradlePlugin)
+        classpath("com.android.tools.build:gradle:7.2.1")
```
Comment (MGaetan89) [inline build.gradle.kts:29]: Isn't that what's already on line 24?

**Comment by JolandaVerhoef** [comment]
Closing this as several changes are already WAI.

**Pull request closed without merging**